### PR TITLE
Use autocomplete for the submission tags

### PR DESCRIPTION
**Pull request by marcoacierno**

### CODE REVIEW

File: backend/api/submissions/forms.py [hunk 1]
```diff
@@ -21,7 +20,7 @@ class SendSubmissionForm(ContextAwareModelForm):
     )
 
     tags = forms.ModelMultipleChoiceField(
-        queryset=SubmissionTag.objects.all(), to_field_name="name", required=False
+        queryset=SubmissionTag.objects.all(), to_field_name="id", required=False
```
Comment (patrick91) [inline backend/api/submissions/forms.py]: 👍

File: backend/api/tests/schema/test_send_submission.py [hunk 2]
```diff
@@ -669,10 +669,11 @@ def test_create_submission_tags(
         audience_levels=("Beginner",),
     )
 
-    SubmissionTag.objects.get_or_create(name="python")
-    SubmissionTag.objects.get_or_create(name="graphQL")
+    python = SubmissionTag.objects.get_or_create(name="python")
+    graphql = SubmissionTag.objects.get_or_create(name="graphQL")
```
Comment (patrick91) [inline backend/api/tests/schema/test_send_submission.py]:
```suggestion
    graphql = SubmissionTag.objects.get_or_create(name="GraphQL")
```

:P

File: backend/submissions/admin.py [hunk 3]
```diff
@@ -23,11 +23,12 @@ class SubmissionAdmin(admin.ModelAdmin):
                 )
             },
         ),
-        ("Details", {"fields": ("elevator_pitch", "abstract", "notes")}),
+        ("Details", {"fields": ("elevator_pitch", "abstract", "notes", "tags")}),
```
Comment (patrick91) [inline backend/submissions/admin.py]: 😍

File: backend/submissions/admin.py [hunk 4]
```diff
@@ -23,11 +23,12 @@ class SubmissionAdmin(admin.ModelAdmin):
...
     )
     list_filter = ("conference", "type", "topic")
     search_fields = ("title", "abstract")
     prepopulated_fields = {"slug": ("title",)}
+    filter_horizontal = ("tags",)
```
Comment (patrick91) [inline backend/submissions/admin.py]: 😍😍😍

File: frontend/src/components/input-tag/index.tsx [hunk 5]
```diff
@@ -50,86 +29,24 @@ export const TagLine: React.SFC<TagLineProps> = ({
     },
   );
 
-  if (loading) {
+  if (loading || error) {
```
Comment (patrick91) [inline frontend/src/components/input-tag/index.tsx]: we should probably throw the error :)

File: frontend/src/components/select/index.tsx [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+        state: { isFocused: boolean },
+      ) => ({
+        ...base,
+        position: "absolute",
+        top: "-10px",
+        right: "-10px",
+        width: "26px",
+        height: "26px",
+        border: "3px solid #000",
+        background: state.isFocused ? "#FFBDAD" : "orange",
```
Comment (patrick91) [inline frontend/src/components/select/index.tsx]: is this orange coming from the theme? 
https://theme-ui.com/hooks

File: frontend/src/components/select/index.tsx [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+        borderRadius: "100%",
+        cursor: "pointer",
+        color: state.isFocused ? "#DE350B" : "#000",
+      }),
+      placeholder: (base: CSSProperties): CSSProperties => ({
+        ...base,
+        fontFamily: "aktiv-grotesk-extended,sans-serif",
```
Comment (patrick91) [inline frontend/src/components/select/index.tsx]: we could inherit this :)

File: frontend/src/components/tickets-form/index.tsx [hunk 8]
```diff
@@ -0,0 +1,100 @@
+import { useQuery } from "@apollo/react-hooks";
```
Comment (patrick91) [inline frontend/src/components/tickets-form/index.tsx]: is this from another PR?

File: frontend/src/components/select/index.tsx (hunk 6 above)
Comment (marcoacierno) [reply frontend/src/components/select/index.tsx]: nope, that's the orange of the browser. Probably worth replacing it the HEX color

File: frontend/src/components/input-tag/index.tsx (hunk 5 above)
Comment (marcoacierno) [reply frontend/src/components/input-tag/index.tsx]: ah yes, I wanted to change it but temporarily used `|| error`

File: frontend/src/components/select/index.tsx (hunk 7 above)
Comment (marcoacierno) [reply frontend/src/components/select/index.tsx]: makes sense, I want to make `react-select` inherit our theme so we do not have to do this (I think)

File: frontend/src/components/tickets-form/index.tsx (hunk 8 above)
Comment (marcoacierno) [reply frontend/src/components/tickets-form/index.tsx]: weird, let me see

Comment (marcoacierno) [reply frontend/src/components/tickets-form/index.tsx]: removed

File: backend/api/tests/schema/test_send_submission.py (hunk 2 above)
Comment (marcoacierno) [reply backend/api/tests/schema/test_send_submission.py]: done 🍡

File: frontend/src/components/input-tag/index.tsx (hunk 5 above)
Comment (marcoacierno) [reply frontend/src/components/input-tag/index.tsx]: done!

File: backend/api/submissions/forms.py (hunk 1 above)
Comment (marcoacierno) [reply backend/api/submissions/forms.py]: 👻

**Pull request merged**